gCoffee/app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,11 @@
         <item>Americano</item>
         <item>Espresso</item>
     </string-array>
+
+    <string name="latte">Latte</string>
```

Comment:
Do we know already how/if it is possible to make this more dynamic?

Reply:
I haven't looked more into this yet